test/spoolchanges.js

```diff
@@ -32,6 +36,18 @@ const db = request.client(`${url}/${dbName}`, { parallelism: 1 });
 
 const seqSuffix = Buffer.alloc(124, 'abc123').toString('base64');
 
+function changes(bufferSize, tolerance) {
+  // Make a pipeline of the spool changes source streams
+  return pipeline(...spoolchanges(db, '/dev/null', bufferSize, tolerance),
+  // add a mapping to string and send to /dev/null as we don't care about the output
+    new MappingStream(JSON.stringify), fs.createWriteStream('/dev/null'))
+    // Historically spool changes itself could return an error, but
+    // now it returns streams to be made a pipeline elsewhere.
+    // As such a conversion takes place in the pipeline level catch, we reproduce
+    // that here by calling the converResponseError function.
```

Comment:
```suggestion
    // that here by calling the convertResponseError function.
```

Reply:
Fixed in 966cdb5b159ed1866ae7ea7abc6ea2fdb25612e7